dom/events/EventListener-handleEvent.html

```diff
@@ -66,6 +66,19 @@
     assert_equals(calls, 2);
 }, "performs `Get` every time event is dispatched");
 
+test(function(t) {
+    var type = "foo";
+    var target = document.getElementById("target");
+
+    var eventListener = function() {};
+    eventListener.handleEvent = function() {
+        assert_unreached("`handleEvent` method should not be called on functions");
```

Comment:
This should be wrapped in `this.step_func()` in general, but for this case, there's a shorter approach with `this.unreached_func()`.

Reply:
4b79865